OptimizelySDK.Tests/ConfigTest/TestPollingProjectConfigManager.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Threading;
+using OptimizelySDK.Config;
+using OptimizelySDK.Logger;
+
+namespace OptimizelySDK.Tests.DatafileManagementTests
+{
+    public class TestPollingData
+    {
+        public int PollingTime { get; set; }
+        public ProjectConfig ConfigDatafile { get; set; }
+        public ProjectConfig ConfigVersioned { 
+            get {
+                if (ConfigDatafile == null)
+                    return null;
+                if (ChangeVersion)
+                    ConfigDatafile.Version = DateTime.Now.Ticks.ToString();
+
+                return ConfigDatafile;
+            }
+        }
+        public bool ChangeVersion { get; set; }
+    }
+
+    public class TestPollingProjectConfigManager : PollingProjectConfigManager
+    {
+        // default.
```

Comment:
Can you elaborate :P

Reply:
Oh this comment is not valid anymore, I was using some default values like delayed time that's why I added default. Going to remove. Thanks :)